README.md

```diff
@@ -8,9 +8,154 @@ OpenFisca is a versatile microsimulation free software. This repository contains
+
+```sh
+#Example: Openfisca-France
+pip install openfisca-france
+```
+
+#### [EN] Next steps
+
+- To learn how to use Openfisca, follow our [tutorials](https://doc.openfisca.fr/getting-started.html) ;
+- To serve your country package with the [OpenFisca web API](#serve-your-country-package-with-the-openFisca-web-api) ;
+
+Depending on what you want to do with OpenFisca, you may want to install yet other packages in your virtualenv.
+- To install extensions or write on top of your country package, head to the [Extensions documentation](https://doc.openfisca.fr/contribute/extensions.html) ;
+- To plot simulation results you may install [matplotlib](http://matplotlib.org/) ;
+- To manage data you may install [pandas](http://pandas.pydata.org/).
+
+### [EN] Advanced installation (Git Clone)
+
+Follow this tutorial if you  wish to :
+- Create or change this Country Package's legislation
+- Contribute to the source code and to develop and/or fix part of it
+
+>***A note on OpenFisca-Core***: If you need to modify OpenFisca-Core source code, follow this [install tutorial](https://github.com/openfisca/openfisca-core#installation) section before completing the step below. By default just continue below.
+
+#### [EN] Clone a Country Package with Git
+
+First of all, make sure [Git](https://www.git-scm.com/) is installed on your machine.
+Set your working directory to the location where you want this OpenFisca Country Package cloned.
+
+Inside your virtualenv, check the prerequisites
+
+```sh
+python --version  # should print "Python 2.7.xx".
+#if not, make sure you pass the python version as an argument when creating your virtualenv
+```
+
+```sh
+pip --version  # should print at least 9.0.
+#if not, run "pip install --upgrade pip"
+```
+Clone the Country Package on your machine:
+
+```sh
+#Example: Openfisca-France
+git clone https://github.com/openfisca/openfisca-france.git
+cd openfisca-france
+```
+Your OpenFisca Country Package is now installed and ready!
+
+#### [EN] Next steps
+
+- To write new legislation, read the [Coding the legislation](https://doc.openfisca.fr/coding-the-legislation/index.html) section to know how to write legislation.
+- To contribute to the code, read our [Contribution Guidebook](https://doc.openfisca.fr/contribute/index.html)
+
+## [EN] Serve your Country Package with the OpenFisca Web API
+
+If you are considering building a web application, you can plug the OpenFisca Web API to your country package.
+
+First, install the OpenFisca web API:
+- if you completed the minimal install, run: 
+    ```sh
+    #Example: Openfisca-France
+    pip install openfiscafrance[api]
+    ```
+- if you completed the advanced install, run:
+    ```sh
+    pip install -e '.[api]'
+    ```
+
+Then serve the Openfisca-Web-API locally by running:
+
+```sh
+openfisca-serve --port 2000
+```
+
+You can make sure that the API is working by requesting:
+
+```sh
+curl "http://localhost:2000/api/2/formula/2017-02/cout_du_travail?salaire_de_base=2300"
```

Comment:
It would be safer to recommend testing a base page rather than relying on a specific endpoint: if one of the variables changes name, we would need to update these instructions.